chapters/planning-data-work.tex

```diff
@@ -151,162 +469,185 @@ \subsection{Code management}
-and because of the specificity with which code depends on file structure,
-you will be able to enforce better practices there than in the project folder.
+One other important advantage of code review if that
+making sure that the code is running properly on other machines,
+and that other people can read and understand the code easily,
+is the easiest way to be prepared in advance for a smooth project handover.
 
+% ----------------------------------------------------------------------------------------------
 \subsection{Output management}
 
-Another task that needs to be discussed with your team is the best way to manage outputs.
-A great number of them will be created during the course of a project,
-from raw outputs such as tables and graphs to final products such as presentations, papers and reports.
+The final task that needs to be discussed with your team is the best way to manage output files.
+A great number of outputs will be created during the course of a project,
+and these will include both raw outputs such as tables and graphs
+and final products such as presentations, papers and reports.
 When the first outputs are being created, agree on where to store them,
-what software to use, and how to keep track of them.
+what softwares and formats to use, and how to keep track of them.
 
 % Where to store outputs
-Decisions about storage of final outputs are made easier by technical constraints.
-As discussed above, Git is a great way to control for different versions of
-plain text files, and sync software such as Dropbox are better for binary files.
-So storing raw outputs in formats like \texttt{.tex} and \texttt{.eps} in Git and
-final outputs in PDF, PowerPoint or Word, makes sense.
-Storing plain text outputs on Git makes it easier to identify changes that affect results.
+Decisions about storage of outputs are made easier by technical constraints.
+As discussed above, version control systems like Git are a great way to manage
+plaintext files, and sync softwares such as Dropbox are better for binary files.
+Outputs will similarly come in these two formats, depending on your software.
+Binary outputs like Excel files, PDFs, PowerPoints, or Word documents can be kept in a synced folder.
+Raw outputs in plaintext formats like \texttt{.tex} and \texttt{.eps}
+can be created from most analytical software and managed with Git.
+Tracking plaintext outputs with Git makes it easier to identify changes that affect results.
 If you are re-running all of your code from the master when significant changes to the code are made,
 the outputs will be overwritten, and changes in coefficients and number of observations, for example,
-will be highlighted.
+will be highlighted for you to review.
+In fact, one of the most effective ways to check code quickly
+is simply to commit all your code and outputs using Git,
+then re-run the entire thing and examine any flagged changes in the directory.
```

Comment:
This sentence only makes sense if you already understand Git:
> In fact, one of the most effective ways to check code quickly is simply to commit all your code and outputs using Git, then re-run the entire thing and examine any flagged changes in the directory.

But I think you have already made the point this sentence is making in preceding sentence. Currently I think the quoted sentence only repeats a point already made, but int a way that might confuse someone new to Git. Two ideas, use neither, either or both. 
1. make it explicit in the sentence before that it is Git highlighting _will be highlighted by Git for you to review_
2. Update this last sentence to say that in DIME Analytics we just exactly this when reviewing code.

Reply:
c71b4ba071c4add8739dbcd88b55ab2861aa79d3